topo_descriptors/topo.py

```diff
@@ -99,10 +99,16 @@ def tpi(dem, size, sigma=None):
 
     if sigma:
         dem = ndimage.gaussian_filter(dem, sigma)
+    conv_fn = lambda a: ndimage.convolve(
+        a, kernel, mode="constant", cval=np.nan, origin=-1
+    )
 
-    conv = signal.convolve(
-        dem, kernel, mode="same"
-    )  # ndimage.convolve(dem, kernel, mode='reflect')
+    if isinstance(dem.data, da.Array):
+        print("dask")
```

Comment:
is this print (and the one few lines below too) necessary? In case, I'd rather use the logger module

Reply:
Oops, I totally forgot to delete this. Thanks for noticing 😉